### PR TITLE
[dvsim] Separate publish report from dvsim flow [PART3]

### PR DESCRIPTION
**Pull request by cindychip**
Small fixes:
1). Add a missing symlink for publish.html.
2). Skip `clean_odirs` if the directory already exists.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

File: util/dvsim/FlowCfg.py [hunk 1]
```diff
@@ -403,43 +403,46 @@ def gen_results(self, results):
...
+            self.write_results_html("summary.html", item.results_summary_md)
         self.gen_email_html_summary()
 
     def gen_results_summary(self):
         '''Public facing API to generate summary results for each IP/cfg file
         '''
         return
 
-    def write_results_html(self):
+    def write_results_html(self, results_name, results_md):
```
Comment (sriyerg) [inline util/dvsim/FlowCfg.py:422]: How about:
```suggestion
    def write_results_html(self, filename, text_md):
```

File: util/dvsim/FlowCfg.py [hunk 2]
```diff
@@ -403,43 +403,46 @@ def gen_results(self, results):
...
         # Keep up to 2 weeks results.
-        clean_odirs(odir=self.results_path, max_odirs=14)
+        # If path exists, skip clean_odirs and directly update the files in the existing path.
+        if not (os.path.exists(self.results_path)):
+            clean_odirs(odir=self.results_path, max_odirs=14)
         mk_path(self.results_path)
         mk_path(self.symlink_path)
 
+        # Prepare results and paths.
+        results_html = md_results_to_html(self.results_title, self.css_file,
```
Comment (sriyerg) [inline util/dvsim/FlowCfg.py:435]:
```suggestion
        text_html = md_results_to_html(self.results_title, self.css_file,
```

**Review by sriyerg — APPROVED** [review]
LGTM, save for 2 nits. Thanks!

File: util/dvsim/FlowCfg.py (hunk 1 above)
Comment (cindychip) [reply util/dvsim/FlowCfg.py:422]: Thank you Sri for the suggestion. This generic name is better.

File: util/dvsim/FlowCfg.py (hunk 2 above)
Comment (cindychip) [reply util/dvsim/FlowCfg.py:435]: Updated as well. Thanks Sri for reviewing.

**Review by sriyerg — APPROVED** [review]

**Pull request merged**